packages/tenant-process/test/utils.ts

```diff
@@ -196,6 +207,13 @@ export const addOneTenant = async (
   await writeInReadmodel(tenant, tenants);
 };
 
+export const addOneAttribute = async (
+  attribute: AttributeReadmodel,
+  attributes: AttributeCollection
+): Promise<void> => {
+  await writeInReadmodel(attribute, attributes);
+};
+
```

Comment:
I think we could avoid to wrap this in a function.
Also, we could remove the `addOneAgreement` and `addOneEService` wrappers (maybe in a different PR). The only one needed is `addOneTenant` because it performs two operations.

Reply:
Makes sense! Thanks